core/domain/exp_jobs_one_off.py

```diff
@@ -831,3 +831,63 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class EmptySubtitledHtmlAuditJob(jobs.BaseMapReduceOneOffJobManager):
+    """One off job to audit explorations having empty subtitled html and outputs
+    the content_id of such object.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+
+        exploration = exp_services.get_exploration_from_model(item)
+        exp_rights = rights_manager.get_exploration_rights(item.id)
+
+        state_name_to_content_id_list = {}
```

Comment:
I am not sure but would it be better to add this as a method for Exploration class in `exp_domain.py` which can be used to obtain content ids of all fields with empty html. (just as we have a [method](https://github.com/oppia/oppia/blob/develop/core/domain/exp_domain.py#L3039) defined to obtain all html strings).

It will be useful only if we will need this method for purposes other than auditing the empty html.

Reply:
I need content ids also the all_html_method returns html from customization agrs of few interactions. (Like Multiple choices etc.)

Reply:
Ya, I was suggesting to have a method that returns content ids but I am not sure that it will be useful further or not.

Reply:
Oh, sorry, I misunderstood your last comment. Thanks for the explanation! :)

Nope, this won't be of any use as we need those content ids which have empty html, and we are planning to add validation for exploration to have "non-empty html".

Reply:
Ah, alright then!